contentcuration/contentcuration/static/js/edit_channel/__tests__/ContentNodeCollection.spec.js

```diff
@@ -0,0 +1,107 @@
+const Backbone = require('backbone');
+
+const ContentNodeCollection = require('../models').ContentNodeCollection;
+
+function exampleContentNode() {
```

Comment:
Would it make sense to start consolidating collections of these kinds of things to be reused in lots of tests?

Reply:
Yes!